css/header-yellow.scss

```diff
@@ -114,51 +155,72 @@
     }
 }
 
-.header-yellow__text {
+.header-yellow__layout {
     color: $color-RO_donkerblauw;
     // See .content with negative margin-top
-    padding: 0 0 2em 1.38em;
+    padding: 0 0 10em 1.38em;
     position: relative;
-    z-index: 2;
 
     @media (min-width: 680px) {
-        padding: 0 0 3.5em 3em;
+        padding: 0 0 15em 4.0em;
     }
 
     @media (min-width: 960px) {
-        padding: 0 0 5em 2.6em;
-        margin-right: 424px;
-        min-height: 430px;
+        padding: 0 0 10em 2.6em;
     }
+}
 
-    h1 {
+.header-yellow__layout__middle {
+    @media (min-width: 960px) {
         position: relative;
-        top: 1.5em;
-        margin: 0.5em 0 0 0;
-        font-size: 1.62em;
-        line-height: 1.16em;
-        width: 50%;
-
-        @media (min-width: 680px) {
-            font-size: 3em;
-            top: 0.2em;
-        }
+        padding: 20px 0 0 200px;
+    }
+}
 
-        @media (min-width: 960px) {
-            top: 1.0em;
-            margin-right: 9rem;
-        }
+.header-yellow__text {
+    position: relative;
+    margin-bottom: 1.5em;
+    font-size: 28px;
+    line-height: 1.16em;
+    max-width: 350px;
+
+    @media (min-width: 680px) {
+        max-width: 700px;
+        padding-right: 0.5em;
+        font-size: 3em;
+        margin-bottom: 0.5em;
     }
 
-    h2 {
-        margin: 0.38em 0 2.3em 0;
-        font-size: 1.38em;
-        line-height: 1.2em;
-        font-weight: 400;
+    @media (min-width: 960px) {
+        width: 100%;
+    }
+}
 
-        @media (min-width: 680px) {
-            margin-bottom: 1.62em;
-            font-size: 2em;
-        }
+.header-yellow__logo-badge {
+    width: 100px;
+    margin: 0 0 1.5em;
+    
+    @media (min-width: 680px) {
+        margin: 1.5em 0;
     }
+
+    @media (min-width: 960px) {
+        width: 132px;
+        position: relative;
+        float: left;
+    }
+}
+
+.yellow__layout__faq-link {
+  position: absolute;
+  top: 38px;
+  right: 230px;
+  min-width: 118px;
+  font-weight: 800;
+  font-size: 20px;
+  display: none;
```

Comment:
Zo is de link niet meer beschikbaar voor gebruikers met _low vision_ of mobiele apparaten. Kan de link niet op een andere plek terugkomen?

Reply:
Deze suggestie heb ik doorgegeven. Op dit moment ligt dit nog open.